components/builder-depot/src/server.rs

```diff
@@ -49,7 +50,9 @@ use super::Depot;
 use config::Config;
 use error::{Error, Result};
 
+define_metrics!();
```

Comment:
I'm sure this is my own Rust ignorance showing, but I don't understand why it's advantageous to use a macro for this, instead of just defining the `Metrics` struct right here.